instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java

```diff
@@ -121,6 +128,34 @@
     verify(span).remoteIpAndPort("1.2.3.4", 5555);
   }
 
+  @Test
+  @TestCaseName("remoteServiceName should be '{1}'")
+  @Parameters({
+    "?zipkinServiceName=myDatabase&foo=bar, myDatabase",
+    "?zipkinServiceName=my_database&foo=bar, my_database",
+    "?zipkinServiceName=my-database&foo=bar, my-database",
+    "?zipkinServiceName=my-database-1&foo=bar, my-database-1",
+    "?zipkinServiceName=my.database&foo=bar, my.database",
+    "?zipkinServiceName=my.database&foo=bar, my.database",
+    "?zipkinServiceName=my-database:5432&foo=bar, my-database:5432",
+    "?zipkinServiceName=my-database@localhost&foo=bar, my-database@localhost",
+    "?zipkinServiceName=my-database, my-database",
+    "?zipkinServiceName=my-database-1&zipkinServiceName=my-database-2, my-database-1",
+    "?zipkinServiceName=, default",
```

Comment:
empty should coerce to null, not "default" IMHO

Reply:
bear in mind that this parameter is an override for `remoteServiceName`, not providing it just results in the default (the database name) used, that's the pre-existing behaviour.